macaroonbakery/discharge.py

```diff
@@ -178,9 +178,11 @@ def _check_need_declared(ctx, cav_info, checker):
     need_declared = arg[0:i].split(',')
     for d in need_declared:
         if d == '':
-            raise bakery.VerificationError('need-declared caveat with empty required attribute')
+            raise bakery.VerificationError('need-declared caveat with empty '
```

Comment:
FWIW we're not enforcing a line-length limit any more. Personally I think that not splitting the line here is a bit easier to read (ditto below). I'd suggest using `tox -e lint` rather than running the linter directly.